src/components/Dropdown/Dropdown.stories.tsx

```diff
@@ -11,19 +11,20 @@ import Label from '../Label';
 import { colors } from '../../index';
 
 const generateCityList = (amount: number): OptionProps[] => {
-  const finalData = [];
+  const tempData: string[] = [];
 
   for (let i = 0; i < amount; i += 1) {
-    const item = address.city();
-    finalData.push({
-      id: item.toLowerCase(),
-      optionValue: item,
-    });
+    let item = address.city();
+    while (tempData.includes(item)) item = address.city();
```

Comment:
I'm surprised prettier didn't add curly brackets and break this to a separate line

Reply:
Yeah, wondered the same. Looks like I forgot to push `item` to the array anyway, so I can format it with the fix.